packages/dds/tree/src/test/utils.ts

```diff
@@ -113,12 +113,15 @@ export class TestTreeProvider {
      * Give this {@link TestTreeProvider} the ability to summarize on demand during a test by creating a summarizer
      * client for the container at the given index. This must be called before any trees submit any edits, or else a
      * different summarizer client might already have been elected.
-     * @param index - the container that will spawn the summarizer client
      * @returns a function which will cause a summary to happen when awaited. May be called multiple times.
      */
-    public async enableManualSummarization(index = 0): Promise<() => Promise<void>> {
-        assert(index < this.trees.length, "Index out of bounds: not enough trees");
-        const summarizer = await createSummarizer(this.provider, this.containers[index]);
+    public async enableManualSummarization(): Promise<() => Promise<void>> {
+        // TODO: support manual summarization with multiple trees once bug #1876 is fixed.
+        assert(
+            this.trees.length === 1,
+            "Manual summarization cannot be enabled for multiple trees.",
```

Comment:
It's still fine to have multiple trees if you create those additional trees *after* calling `enableManualSummarization`, right?
Assuming that's correct (and it seems to be since I'm relying on it), we should mention that in the message. Otherwise you get the wrong idea as to what is really possible or not.

Reply:
Changes made such that the `create()` method now takes in a `summarizeOnDemand` parameter, and creates the summarizer after creating the first tree (subsequent trees are created afterwards).